client/v2_2/docker_image_.py

```diff
@@ -352,12 +405,12 @@ def _gzipped_content(self, name):
   def _populate_manifest_and_blobs(self):
     """Populates self._manifest and self._blob_names."""
     # TODO(user): Update mimes here for oci_compat.
+    config_blob = 'sha256:' + hashlib.sha256(self.config_file()).hexdigest()
```

Comment:
config_blob = util.Digest(self.config_file())

Reply:
Might want genericize the hashing util to not refer to just manifests...? @mattmoor